configs/webpack-config-compass/src/webpack-plugin-multicompiler-progress.ts

```diff
@@ -43,7 +43,7 @@ function progressHandler(
   const padStartMsg = Math.max(
     ...(Array.from(bars.values()).map((bar) =>
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      bar ? (bar as any).payload.msg.trim().length : 0
+      bar ? ((bar as any).payload.msg || '').trim().length : 0
```

Comment:
what is this for?

Reply:
The package-lock.json changes caused some webpack plugin to update and now payload.msg is sometimes not a string (undefined or null? can't remember) and this ONLY happens in CI causing every package-compass task to fail. Couldn't reproduce it locally, just worked around it. webpack build progress still seems to work. 🤷